gax/src/test/java/com/google/api/gax/rpc/testing/FakeBatchableApi.java

```diff
@@ -82,6 +83,10 @@ public int hashCode() {
       result = 31 * result + ints.hashCode();
       return result;
     }
+
+    public LabeledIntList toBuilder() {
```

Comment:
a LabeledIntList is a builder for itself? Maybe you meant clone()?

Reply:
Yes, I meant a `clone()`. Have now added a custom `clone` method.